impl/src/main/java/io/kroxylicious/testing/kafka/common/KafkaClusterFactory.java

```diff
@@ -97,6 +97,10 @@ private static KafkaClusterExecutionMode getExecutionMode(KafkaClusterConfig clu
                 clusterConfig.getExecMode() == null ? KafkaClusterExecutionMode.IN_VM : clusterConfig.getExecMode());
     }
 
+    private static String getKafkaVersion(KafkaClusterConfig clusterConfig) {
+        return System.getenv().getOrDefault(KAFKA_VERSION, clusterConfig.getKafkaVersion() == null ? "latest" : clusterConfig.getKafkaVersion());
```

Comment:
I've changed `clusterConfig.getKafkaVersion()` so that it nevers return nulls.
We should now never go down the "latest" path unless someone does an explicit `@Version("latest")`.